pyrdp/player/LiveWindow.py

```diff
@@ -37,15 +42,22 @@ def onConnection(self) -> asyncio.Protocol:
 
     def createLivePlayerTab(self):
         tab = LiveTab()
-        tab.connectionClosed.connect(self.onConnectionClosed)
+        tab.connectionRenameTab.connect(self.renameLivePlayerTab)
         self.addTab(tab, "New connection")
-        self.setCurrentIndex(self.count() - 1)
+
+        if self.options.get("focusNewTab"):
+            self.setCurrentIndex(self.count() - 1)
+
+        self.updateCountSignal.emit()
         self.queue.put(tab)
 
-    def onConnectionClosed(self, tab: LiveTab):
+    def renameLivePlayerTab(self, tab: LiveTab, name: str, closed: bool):
         index = self.indexOf(tab)
-        text = self.tabText(index)
-        self.setTabText(index, text + " - Closed")
+        if closed:
```

Comment:
Proof that they are separate: you check if one argument is present, otherwise you check the other argument.